distutils/_msvccompiler.py

```diff
@@ -178,15 +178,24 @@ def _find_exe(exe, paths=None):
     return exe
 
 
-# A map keyed by get_platform() return values to values accepted by
```

Comment:
I noticed we lost this definition of `PLAT_TO_VCVARS`. It might be nice to bring that back somehow, but I'm struggling with how, because it is now defined twice. I'm going to try some things.

Reply:
For now I restored the comment... but yes IDEs would be confused 🤔

Reply:
VSCode seems to not sure anything anyway even with the current main branch, huh.

Reply:
IDEs have to be able to handle multiple conditional definitions in Python - it's a fundamental part of the language (it's even supported in type stub files). If they don't, file bugs with the IDEs.